mutiny-wasm/src/models.rs

```diff
@@ -889,3 +889,39 @@ impl From<nostr::nwc::PendingNwcInvoice> for PendingNwcInvoice {
         }
     }
 }
+
+// This is the NodeIdentity that refer to a specific node
+// Used for public facing identification.
+#[derive(Serialize, Deserialize, Clone, Eq, PartialEq)]
+#[wasm_bindgen]
+pub struct Plan {
+    id: u8,
+    amount_sat: u64,
+}
+
+#[wasm_bindgen]
+impl Plan {
+    #[wasm_bindgen(getter)]
+    pub fn value(&self) -> JsValue {
+        JsValue::from_serde(&serde_json::to_value(self).unwrap()).unwrap()
+    }
+
+    #[wasm_bindgen(getter)]
+    pub fn id(&self) -> u8 {
+        self.id.clone()
+    }
+
+    #[wasm_bindgen(getter)]
+    pub fn amount_sat(&self) -> u64 {
+        self.amount_sat
+    }
```

Comment:
you don't need these and can just make the values `pub`, we only need to do this for Strings